app/views/orders/new.html.erb

```diff
@@ -0,0 +1,71 @@
+<h1>Detalles de la entrega <%= @product.name %></h1>
+
+<h2>Destinatario <%= @product.name %></h2>
+
+<div>
+  <%= form_with model: Order.new, url: orders_path, method: :post, local: true do |form| %>
+    <p>Destinatario</p>
+    <%= form.hidden_field :product_id, value: @product.id %>
+    <div>
+      <%= form.label :quantity, "Cantidad" %>
+      <%= form.number_field :quantity, in: 1..10  %>
+    </div>
+    <div>
+      <%= form.label :bussines_name, "Razon social" %>
+      <%= form.text_field :bussines_name %>
+    </div>
+    <div>
+      <%= form.label :rut, "RUT" %>
+      <%= form.text_field :rut %>
+    </div>
+    <div>
+      <%= form.label :receiver_full_name, "Nombre y apellido" %>
+      <%= form.text_field :receiver_full_name %>
+    </div>
+    <div>
+      <%= form.label :receiver_contact_number, "Número de contacto" %>
+      <%= form.text_field :receiver_contact_number %>
+    </div>
+    <div>
+      <%= form.label :ship_date, "Fecha de entrega" %>
+      <%= form.date_select :ship_date %>
+    </div>
+    <div>
+      <%= form.label :ship_time, "Horarios de entrega" %>
```

Comment:
The designs imply that we could have many shipment destinations. I'd say we should model that by adding a model for the destination and adding a nested form for the destination attributes

Reply:
would it be better to add a json as the 'addresses' attribute instead of another model that is gonna save at least one address per order record?

Reply:
Can you check this [commit](https://github.com/AguirreCampiNicolas/onboarding/commit/bfbdb0f58ac1fd72e8439e80df75f8100d05f424)? I think it is okay now.